jpa/eclipselink.jpa.test/src/it/java/org/eclipse/persistence/testing/tests/jpa22/advanced/StoredProcedureQueryTestSuite.java

```diff
@@ -1011,7 +1012,7 @@ public void testStoredProcedureQuerySysCursor_Positional() {
      * Tests a StoredProcedureQuery using a system cursor. Also tests
      * getParameters call AFTER query execution.
      */
-    public void testStoredProcedureQuerySysCursor2() {
+    public void testStoredProcedureQuerySysCursor_ResultList_Named() {
```

Comment:
Everything looks good in the tests so far except I think this test needs updated to use `getResultList()` like `testStoredProcedureQuerySysCursor_ResultList_Positional` does. This version still uses the old `getOutputParameterValue("")` operation that is tested by `testStoredProcedureQuerySysCursor_Named`